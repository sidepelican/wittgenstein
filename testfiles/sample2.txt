hoge
fuga

// WITTGENSTEIN_BEGIN `head main.go | grep package`
must be overriden here
// WITTGENSTEIN_END
1234567890
# WITTGENSTEIN_BEGIN `echo hello && echo world`
must be overriden here
# WITTGENSTEIN_END

# WITTGENSTEIN_BEGIN `cat aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.bbbbbbbbbbbb`
must be overriden here
# WITTGENSTEIN_END