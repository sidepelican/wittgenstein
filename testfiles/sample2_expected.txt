hoge
fuga

// WITTGENSTEIN_BEGIN `head main.go | grep package`
package main
// WITTGENSTEIN_END
1234567890
# WITTGENSTEIN_BEGIN `echo hello && echo world`
hello
world
# WITTGENSTEIN_END

# WITTGENSTEIN_BEGIN `cat aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.bbbbbbbbbbbb`
cat: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.bbbbbbbbbbbb: No such file or directory
# WITTGENSTEIN_END
